rocketpy/Function.py

```diff
@@ -73,6 +76,22 @@ def __init__(
         self.last_interval = 0
         # Set source
         self.setSource(source)
+        #  Set function title
+        if title:
+            self.setTitle(title)
+        else:
+            if self.__domDim__ == 1:
+                self.setTitle(
+                    self.__outputs__[0].title() + " x " + self.__inputs__[0].title()
+                )
+            elif self.__domDim__ == 2:
+                self.setTitle(
+                    self.__outputs__[0].title()
+                    + " x "
+                    + self.__inputs__[0].title()
+                    + " x "
+                    + self.__inputs__[1].title()
+                )
```

Comment:
I think the way you added the title parameter is really good and actually does not really need the `setTitle` method. 
I'd suggest just deleting it and where you used here `self.setTitle( ... )`  just do `self.title = ... ` instead